packages/@o3r/eslint-config/schematics/ng-add/index.spec.ts

```diff
@@ -36,13 +36,24 @@ describe('ng add eslint-config', () => {
     await runner.runSchematic('ng-add', {}, Tree.empty());
     expect(setupDependenciesMock).toHaveBeenCalledWith(expect.objectContaining({
       dependencies: expect.objectContaining({
+        '@eslint-community/eslint-plugin-eslint-comments': expect.objectContaining({}),
+        '@eslint/js': expect.objectContaining({}),
         '@o3r/eslint-plugin': expect.objectContaining({}),
         '@stylistic/eslint-plugin': expect.objectContaining({}),
+        '@typescript-eslint/parser': expect.objectContaining({}),
         'angular-eslint': expect.objectContaining({}),
         eslint: expect.objectContaining({}),
+        'eslint-import-resolver-node': expect.objectContaining({}),
+        'eslint-import-resolver-typescript': expect.objectContaining({}),
+        'eslint-plugin-import': expect.objectContaining({}),
+        'eslint-plugin-import-newlines': expect.objectContaining({}),
+        'eslint-plugin-jasmine': expect.objectContaining({}),
```

Comment:
Do we really expect jasmine to be imported with jest?

Reply:
We don't use it so I removed it :)